src/demo/demoUtils.ts

```diff
@@ -295,6 +298,15 @@ export const getInitSdkOptions = (): SdkOptions => {
   const customUI =
     queryParamToValueString.customisedUI === 'true' ? customUIConfig : undefined
 
+  const crossDeviceClientIntroProductName =
+    queryParamToValueString.crossDeviceClientIntroCustomProductName === 'true'
```

Comment:
In the contributor's file it says that this query string is expecting a string, not a boolean

Reply:
Good catch! Confused it with the integration documentation in the contributing file.